htmresearch/algorithms/apical_dependent_temporal_memory.py

```diff
@@ -386,9 +408,16 @@ def _calculateLearning(self,
 
   @staticmethod
   def _calculateSegmentActivity(connections, activeInput, connectedPermanence,
-                                activationThreshold, minThreshold):
+                                activationThreshold, minThreshold,
+                                reducedBasalThreshold,
+                                reducedBasalThresholdCells = ()):
     """
-    Calculate the active and matching segments for this timestep.
+    Calculate the active and matching basal segments for this timestep.
+
+    The difference with _calculateApicalSegmentActivity is that cells
+    with active apical segments (collected in reducedBasalThresholdCells) have
+    a lower activation threshold for their basal segments (set by
+    reducedBasalThreshold parameter).
```

Comment:
You could just change this docstring back to "Calculate the active and matching segments for this timestep."